swarm/pss/pss_test.go

```diff
@@ -198,10 +199,15 @@ func TestCache(t *testing.T) {
 		t.Fatalf("message %v should NOT have EXPIRE record in cache but checkCache returned true", msgtwo)
 	}
 
-	time.Sleep(pp.CacheTTL)
+	time.Sleep(pp.CacheTTL*2)
+
```

Comment:
The test should check having an unexpired _and_ expired entry in the cache at the same time, then clean, then check if only the one is gone.

Reply:
Good idea.  I've included this test case.